initializers/object_permissions.yml

```diff
@@ -0,0 +1,22 @@
+#- name: all.ro
+#  description: 'Read Only for All Objects'
+#  enabled: true
+#  # object_types: all
+#  groups:
+#    - applications
+#    - readers
+#  actions:
+#    - view
+#- name: all.rw
+#  description: 'Read/Write for All Objects'
+#  enabled: true
+#  # object_types: all
```

Comment:
Why is this double-commented? Is `object_types` an array or what other values besides `all` are possible?

Reply:
same as above about object_permissions.yml